### PR TITLE
Canadian English Language

### PR DESCRIPTION
**Pull request by thomaswong66**

### CODE REVIEW

**Comment by jim-parry** [comment]
1) Why the other, closed PRs?
2) With a sub-locatle, like this, the ONLY translations that would be part of it are those that differ from the parent locale. You have included the whole thing, making it hard to tell what might differ for en-ca vs en. I took a look, and didn't see anything different :-/

**Comment by thomaswong66** [comment]
I had trouble GPG signing the other PRs so I had to close them. The main changes I made was the spelling of flavor to flavour as well as trademark to trade-mark.

**Comment by jim-parry** [comment]
The keys that have unchanged translation values should be removed.
... which means you still have a few more to do!

**Comment by jim-parry** [comment]
ps the locale would be en-CA

File: app/Language/en-ca/Download.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		<a href="https://github.com/bcit-ci/CodeIgniter/archive/2.2.6.zip" class="nav-link btn-primary" title="Download CodeIgniter 2">
+			<i class="glyphicon glyphicon-save"></i> {downloadButton}
+		</a>
+	</li>
+</ul>
+EOS;
+
+return [
+	'title'				 => 'Download CodeIgniter',
+	'heading'			 => 'CodeIgniter comes in three flavours: CodeIgniter 3 (current), CodeIgniter 4 (future) and CodeIgniter 2 (legacy)',
```
Comment (jim-parry) [inline app/Language/en-ca/Download.php]: This is the only translation key that I see differing from "en".
The other keys, if not different, don't belong.

File: app/Language/en-ca/Legal.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	'marksLink'			 => '[several versions of the CodeIgniter logo](/data/ci-logo.zip)',
+	'logoTitle'			 => 'Privacy Policy',
+	'logoText'			 => $logoGuidelines,
+	'logoLink'			 => '[several versions of the CodeIgniter 4 logo](/data/ci4-logo.zip)',
+	'thirdPartyTitle'	 => 'Website data',
+	'thirdPartyText'	 => $thirdParty,
+	'externalLinksTitle' => 'Links to Other Web Sites',
+	'externalLinksText'	 => $externalLinks,
+	'endText'			 => 'We invite you to read BCIT\'s [main privacy page](http://www.bcit.ca/privacy/)'
+];
```
Comment (jim-parry) [inline app/Language/en-ca/Legal.php]: I don't see any text here that differs fro the "en" verison.

Comment (thomaswong66) [reply app/Language/en-ca/Legal.php]: I have replaced "trademark" with "trade-mark" as per canadian english standards.

**Comment by jim-parry** [comment]
My earlier comments about only needing the "different" translations still apply

**Pull request closed without merging**